tasks/67_find-elements/find-elements.js

```diff
@@ -0,0 +1,6 @@
+function findElement(arr, func) {
+  const resIndex = arr.findIndex(func);
```

Comment:
I think the idea of this task is to write this method yourself, not to create a wrapper.